ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/packages/index.tsx

```diff
@@ -84,8 +88,95 @@ const columns = [
       </a>
     ),
   }),
+  columnHelper.display({
+    id: 'moreInfo',
+    header: () => null,
+    size: 50,
+    cell: ({ row }) => {
+      return row.getCanExpand() ? (
+        <Button
+          variant='outline'
+          size='sm'
+          {...{
+            onClick: row.getToggleExpandedHandler(),
+            style: { cursor: 'pointer' },
+          }}
```

Comment:
This object destructuring shouldn't be needed.

Reply:
Probably was a leftover from some more complicated code.